si_ps_conv_fastcorrsApp/Db/kicks.template

```diff
@@ -0,0 +1,219 @@
+record(ao, "out-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(OUT, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP CA")
+}
+
+record(ao, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(OUT, "@strn2curr $(SEC) $(DIP) $(ORI) Current-SP")
```

Comment:
If everything receives a `TEST` prefix, I suppose all `@` expressions would use `$(TEST)$(SEC)` for the first argument